<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWidget</class>
 <widget class="QWidget" name="SetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>272</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="gamePathLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>56</y>
     <width>224</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Game path (Half-Life)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gamePathBrowserButton">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>80</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Browser</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gamePathEdit">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>80</y>
     <width>160</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="executeCmdLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>200</y>
     <width>224</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Execute command:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="executeCmdEdit">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>224</y>
     <width>224</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="gameModLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>152</y>
     <width>224</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mod:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gameModEdit">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>176</y>
     <width>224</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="compilerLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>224</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Compiler</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compilerBrowserButton">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>32</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Browser</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="compilerEdit">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>32</y>
     <width>160</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputPathEdit">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>128</y>
     <width>160</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="outputPathLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>104</y>
     <width>224</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Compile output path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputPathBrowserButton">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>128</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Browser</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
